### DISCUSSION TITLE
[Feature] customizeable date/time position

### DISCUSSION THREAD

**Discussion by AlexStephenLytton** [damongolding/immich-kiosk#101] [Feature request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be great, if the position of time, date, location, etc would be customizeable (like top left, top right, bottom left, bottom right). I use this awsome tool to view my images in an iframe within MagicMirror. I prefer this variant over the immich module because of the background blur tbh. 
In my configuration there are other modules shown in the lower right corner which are overlapping with the time/date/location info provided by immich-kiosk.

Another great improvement (for me) would be changing the order of time/date to date/time

or maybe an option to add a custom.css?

Greetings,

Alex

**Comment by damongolding** [comment]
Hey. 

I'm currently implementing two features that style the UI (layouts and themes) so I think adding such granular options to the UI positions might get a little complicated. 

I have thought about adding a custom css option though. Just have to think of the best way to implement it!

**Reply by AlexStephenLytton** [reply]
> I have thought about adding a custom css option though. Just have to think of the best way to implement it!

Sounds great :) 👍 

This Container is great by any means :)

**Comment by damongolding** [comment]
![localhost_3000_ (2)](https://github.com/user-attachments/assets/1f34ab5c-8207-4989-9744-5c9be7aa7225)

Custom CSS

design is my PASSION 

```css
* {
    border: 1px solid red;
    color: orange;
    background-color: blue;
    box-sizing: border-box;
}

.frame {
    display: none;
}

```

**Reply by AlexStephenLytton** [reply]
👍

**Comment by genebean** [comment]
On a related note, the screenshots show the time on the bottom but it appears on the top for me … is am I missing something config wise to have it on the bottom, or is that a custom css thing?

**Reply by damongolding** [reply]
It was originally in the bottom left but moved as the UI became more complex. I just haven't gotten round to updating the screenshots 😅
